### PR TITLE
update isolated task execution for ansible-runner

### PR DESCRIPTION
**Pull request by ryanpetrello**
u https 3a 2f 2fmedia giphy com 2fmedia 2fy5q7nwkyqapo 2fgiphy

**WARNING: I am about 95 certain that this *doesn't* actually specify the correct bwrap args on the isolated host.**  You gotta figure that one out.

image

image

image

### CODE REVIEW

**Comment by ryanpetrello** [comment]
cc @shanemcd

**Comment by ryanpetrello** [comment]
Also @chrismeyersfsu I have not tested custom venvs on isolated nodes _at all_, but I _think_ they might just work.

File: awx/main/tasks.py [hunk 1]
```diff
@@ -648,14 +649,14 @@ def _wrapped(self, *args, **kwargs):
...
         finally:
-            for p in self.cleanup_paths:
-                try:
-                    if os.path.isdir(p):
-                        shutil.rmtree(p, ignore_errors=True)
-                    elif os.path.exists(p):
-                        os.remove(p)
-                except OSError:
-                    logger.exception("Failed to remove tmp file: {}".format(p))
+            #for p in self.cleanup_paths:
```
Comment (ryanpetrello) [inline awx/main/tasks.py:652]: WHOOPS!  Need to back this out :)

File: tools/docker-isolated/Dockerfile [hunk 2]
```diff
@@ -3,23 +3,20 @@ RUN yum clean all
...
-ADD requirements/requirements_ansible.txt requirements/requirements_ansible_git.txt requirements/requirements_ansible_uninstall.txt requirements/requirements_isolated.txt /tmp/requirements/
 RUN yum -y update && yum -y install curl epel-release
 RUN yum -y update && yum -y install openssh-server ansible mg vim tmux git python-devel python36 python36-devel python-psycopg2 make python-psutil libxml2-devel libxslt-devel libstdc++.so.6 gcc cyrus-sasl-devel cyrus-sasl openldap-devel libffi-devel zeromq-devel python-pip xmlsec1-devel swig krb5-devel xmlsec1-openssl xmlsec1 xmlsec1-openssl-devel libtool-ltdl-devel bubblewrap zanata-python-client gettext gcc-c++ libcurl-devel python-pycurl bzip2
 
 RUN ln -s /usr/bin/python36 /usr/bin/python3
 RUN python36 -m ensurepip
 RUN pip3 install virtualenv
+RUN pip3 install git+https://github.com/ansible/ansible-runner@master#egg=ansible_runner
 WORKDIR /tmp
-RUN make requirements_ansible
```
Comment (ryanpetrello) [inline tools/docker-isolated/Dockerfile:14]: Eh, we should probably actually leave this line in so playbooks have the ansible env around w/ its built-in dependencies.

**Comment by chrismeyersfsu** [comment]
Cherry-picked, thanks

**Pull request closed without merging**